lib/matplotlib/backend_managers.py

```diff
@@ -258,20 +252,18 @@ def add_tool(self, name, tool, *args, **kwargs):
         matplotlib.backend_tools.ToolBase : The base class for tools.
         """
 
-        tool_cls = self._get_cls_to_instantiate(tool)
```

Comment:
forgot to remove the method `_get_cls_to_instantiate`

Reply:
fixed, thanks.